client/post-editor/post-editor.jsx

```diff
@@ -162,6 +160,10 @@ const PostEditor = React.createClass( {
 		}
 	},
 
+	hideNotice: function() {
+		this.setState( { notice: null } );
```

Comment:
Hum, curious, I haven't noticed before. Turns out, using the spread operator on an empty object is exactly the same as doing it on `null`. So this works, but I'll change the other occurence to `null`, for consistency.